charts/narrativelog/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
A more standardized ingress that works well with the Phalanx/RSP auth is https://github.com/lsst-sqre/charts/blob/master/rsp-starter/templates/ingress.yaml